OpenMP.make

```diff
@@ -14,13 +14,19 @@ ifndef TARGET
 define target_help
 Set TARGET to change device (defaulting to CPU).
 Available targets are:
-  CPU NVIDIA
+  CPU NVIDIA AMD
 
 endef
 $(info $(target_help))
 TARGET=CPU
 endif
 
+ifeq ("$(TARGET)","AMD")
+# to override AOMP_GPU: $ export AOMP_GPU=gfx906
+INSTALLED_GPU  = $(shell /opt/rocm/bin/rocm_agent_enumerator | grep gfx9 | grep -v gfx000)
```

Comment:
This fails if you have >1 GPU installed on your system, because it prints out both lines and tries to append them both after "-march=" below. For example, on my test system with two gfx906 GPUs, I see:
```
$ COMPILER=CLANG TARGET=AMD make -f OpenMP.make
clang++ -O3 -std=c++11 -DOMP main.cpp OMPStream.cpp -DOMP_TARGET_GPU -O3 -target x86_64-pc-linux-gnu -fopenmp -fopenmp-targets=amdgcn-amd-amdhsa -Xopenmp-target=amdgcn-amd-amdhsa -march=gfx906 gfx906  -o omp-stream
clang: error: no such file or directory: 'gfx906'
make: *** [OpenMP.make:71: omp-stream] Error 1
```

I'd guess you want something like:
`shell /opt/rocm/bin/rocm_agent_enumerator | grep gfx9 | head -n 1`